src/pages/[platform]/build-a-backend/add-aws-services/rest-api/test-api/index.mdx

```diff
@@ -0,0 +1,152 @@
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+          
+
+## Test the API from the terminal
+
+If Guest users have access to your REST API you can test it from the terminal using curl.
+
+[Curl](https://github.com/curl/curl) is a command-line tool that lets you transfer data to and from a server using various protocols.
+
+> Curl is available in many distributions including Mac, Windows and Linux. Follow the install instructions in the [docs](https://curl.haxx.se/docs/install.html).
+
+<BlockSwitcher>
+<Block name="Mac and Linux">
+
+### GET method example
+
+```bash title="Terminal" showLineNumbers={false} 
+curl https://a5b4c3d2e1.execute-api.eu-west-2.amazonaws.com/dev/items
+```
+
+### POST method example
+
+```bash title="Terminal" showLineNumbers={false} 
+curl -H "Content-Type: application/json" -d '{"name":"item-1"}' https://a5b4c3d2e1.execute-api.eu-west-2.amazonaws.com/dev/items
+```
+
+</Block>
+<Block name="Windows">
+
+### GET method example
+
+```bash title="Terminal" showLineNumbers={false} 
+curl https://a5b4c3d2e1.execute-api.eu-west-2.amazonaws.com/dev/items
+```
+
+### POST method example
+
+```bash title="Terminal" showLineNumbers={false} 
+curl -H "Content-Type: application/json" -d {\"name\":\"item-1\"} https://a5b4c3d2e1.execute-api.eu-west-2.amazonaws.com/dev/items
+```
+
+</Block>
+</BlockSwitcher>
+
+<Callout warning="true">
+
+**Warning:** Testing methods using production endpoints may result in changes to resources that cannot be undone.
+
+</Callout>
+
+## Test the API with API Gateway console
+
+Let's test your new REST API using the route below with HTTP Method `GET` and path `/items?limit=10` which includes a `limit` query string parameter.
+
+```console
+GET /items?limit=10
+```
+
+<Callout warning="true">
+
+**Warning:** Testing methods with the API Gateway console may result in changes to resources that cannot be undone.
+
+</Callout>
+
+- Sign in to the [API Gateway console](https://console.aws.amazon.com/apigateway).
+- Choose the `myRestApi` REST API.
+- In the Resources pane, choose the method you want to test. Select `GET` right under `/items`.
+
+```console
+/                        
+ |_ /items               Main resource. Eg: /items
+   GET                   Methods
+   DELETE
+   PUT
+   POST
+   OPTIONS               Allow pre-flight requests in CORS by browser
+    |_ /{proxy+}         Proxy resource. Eg: /items/, /items/id, items/object/{id}
+    ANY                  Includes methods: DELETE, GET, HEAD, OPTIONS, PATCH, POST, PUT
+    OPTIONS              Allow pre-flight requests in CORS by browser
+```
+
+- In the Method Execution pane, select **TEST**. Choose the `GET` method and add `limit=10` to the query string `{items}` field.
+
+- Choose **Test** to run the test for `GET /items?limit=10`. The following information will be displayed: request, status, latency, response body, response headers and logs.
```

Comment:
```suggestion
1. Sign in to the [API Gateway console](https://console.aws.amazon.com/apigateway)
2. Choose the `myRestApi` REST API
3. In the Resources pane, choose the method you want to test. Select `GET` right under `/items`.

  ```console
  /                        
  |_ /items               Main resource. Eg: /items
    GET                   Methods
    DELETE
    PUT
    POST
    OPTIONS               Allow pre-flight requests in CORS by browser
      |_ /{proxy+}         Proxy resource. Eg: /items/, /items/id, items/object/{id}
      ANY                  Includes methods: DELETE, GET, HEAD, OPTIONS, PATCH, POST, PUT
      OPTIONS              Allow pre-flight requests in CORS by browser
  ```

4. In the Method Execution pane, select **TEST**. Choose the `GET` method and add `limit=10` to the query string `{items}` field.
5. Choose **Test** to run the test for `GET /items?limit=10`. The following information will be displayed: request, status, latency, response body, response headers and logs.
```

since we're listing steps this should be represented as an ordered list